05/index.html

```diff
@@ -0,0 +1,150 @@
+            id="menu-switcher"
+          />
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a
+                href="/"
+                class="nav__link"
+              ></a
+              >Home
+            </li>
+            <li class="nav__item">
+              <a
+                href="#"
+                class="nav__link"
+              ></a
+              >Link1
+            </li>
+            <li class="nav__item">
+              <a
+                href="#"
+                class="nav__link"
+              ></a
+              >Link2
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+    <main>
+      <section class="section">
+        <div class="wrapper">
+          <article class="article">
+            <p class="article__content">
+              Lorem, ipsum dolor sit amet consectetur adipisicing elit.
+              Necessitatibus itaque incidunt ea iste molestiae beatae iure est
+              at perferendis optio! Pariatur commodi quo sapiente aut est
+              deserunt voluptate, soluta rerum inventore veritatis odio aliquam
+              facere iste ex corrupti perspiciatis ab labore aperiam, similique
+              voluptates, cumque delectus adipisci? Aliquid, expedita? Sunt ex,
+              neque accusantium corrupti quod nesciunt dolorum, labore maxime
+              inventore totam iusto! Illum, alias laborum. Praesentium, nesciunt
+              blanditiis aliquid officia veniam consequuntur reiciendis
+              deserunt, expedita suscipit labore libero architecto odio quod
+              mollitia molestias rem quas iure. Pariatur aliquam illum veniam
+              exercitationem aut necessitatibus vel maxime quo obcaecati sunt
+              mollitia blanditiis, quisquam assumenda. Tempore iste eveniet sed
+              dolore laborum maxime odit consequatur sequi exercitationem et
+              dignissimos reiciendis consequuntur doloremque facilis, ullam
+              itaque eligendi molestiae! Reprehenderit, porro sunt. Provident
+              quas incidunt veniam rerum suscipit, maiores accusantium. A, id
+              veritatis recusandae alias magni illo illum minima! Commodi
+              aspernatur, reiciendis assumenda illo est quod atque delectus,
+              voluptas iure enim amet odio, consequuntur voluptatibus laborum
+              expedita incidunt provident officiis. Commodi, impedit. Vel, harum
+              corrupti.
+
+              <figure class="article__image">
+                <!-- <img
+                  src="https://newevolutiondesigns.com/images/freebies/4k-dog-wallpaper-4.jpg"
+                  alt="dog"
+                  class="article__image"
+                /> -->
+              </figure>
+            </p>
+          </article>
+        </div>
+      </section>
+      <section class="section">
+        <article class="wrapper gallery__wrapper">
+          <figure class="gallery__image image1">
```

Comment:
Skoro mamy `gallery__wrapper` czy `gallery__image` to powinien pojawić sie w kodzie `block` - w tym przypadku idealnym miejsce jest sekcja czyli `<section class="gallery section">`